Views/PollViews.py

```
@@ -96,8 +98,18 @@ async def button_callback(self, interaction:discord.Interaction):
     #Send ephemeral message with current leader
     async def seeCurrentResults(self, interaction:discord.Interaction):
         await deferInt(interaction)
-        self.BVIObject.updateResults()
-        await interaction.followup.send(f"The current leader is {self.BVIObject.winner}\nSee https://bettervoting.com/{self.BVIObject.electionID}/results for more details", ephemeral=True)
+        try:
+            imgID = self.BVIObject.createBar()
+        except(e):
```

Ruff (0.15.9)

[error] Undefined name `e`

(F821)